examples/organizations/delegatedadministrator.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: organizations.aws.upbound.io/v1beta1
+kind: DelegatedAdministrator
+metadata:
+  annotations:
+    meta.upbound.io/example-id: organizations/v1beta1/delegatedadministrator
+    upjet.upbound.io/manual-intervention: "This resource requires permissions to work with organizations, manually tested with a free tier account"
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    accountIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: account
+    region: us-west-1
+    servicePrincipal: config.amazonaws.com
```

Comment:
AFAIR, this config requires this API to be enabled via CLI, a comment would be helpful for future _manual_ testers.

Reply:
I added a comment for this, thank you.